requirements.txt

```diff
@@ -7,3 +7,4 @@ lxml==3.3.6
 django-storages==1.5.2
 boto==2.46.1
 django-model-utils==2.3.1
+Pillow==4.2.1
```

Comment:
@Qubad786 edx-platform is uses [Pillow==3.4](https://github.com/edx/edx-platform/blob/master/requirements/edx/base.txt#L83)